File: lwjgl3/src/main/java/io/github/msameer0/rhythmicrush/lwjgl3/PackMenuAtlas.java
package io.github.msameer0.rhythmicrush.lwjgl3;

import com.badlogic.gdx.tools.texturepacker.TexturePacker;

public class PackMenuAtlas {
    public static void main(String[] args) {
        TexturePacker.Settings settings = new TexturePacker.Settings();
        settings.maxWidth = 2048;
        settings.maxHeight = 2048;
        settings.edgePadding = true; // keep optional padding
        settings.duplicatePadding = true;

        TexturePacker.process(settings,
            "assets/menu", // input folder
            "assets",      // output folder
            "menu"         // atlas name
        );
    }
}

File: lwjgl3/src/main/kotlin/io/github/msameer0/rhythmicrush/lwjgl3/atlases/PackMenuAtlas.kt
package io.github.msameer0.rhythmicrush.lwjgl3.atlases

import com.badlogic.gdx.tools.texturepacker.TexturePacker

object PackMenuAtlas {
    @JvmStatic
    fun main(args: Array<String>) {
        val settings = TexturePacker.Settings()
        settings.maxWidth = 2048
        settings.maxHeight = 2048
        settings.edgePadding = true // keep optional padding
        settings.duplicatePadding = true

        AtlasPackingUtils.packAllQualities(
            "textures_to_put_in_atlases/menu",
            "assets",
            "menu",
            settings
        )
    }
}
